gwswlib/importer.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import logging
+import os
+import csv
+
+from gwswlib.hydx import Hydx, ConnectionNode
+
+logger = logging.getLogger(__name__)
+
+
+def importhydx(hydxpath):
```

Comment:
Zorg ervoor dat je voor dit soort zaken consistent bent in je naamgeving.
Zelf gebruik ik namen zoals:

    xyz_file: geopende file
    xyz_filename: naam van een bestand
    xyz_abspath: absoluut path naar een bestand
    xyz_relpath: relatief path naar een bestand
    xyz_dirname: naam van een directory
    xyz_dirpath: absoluut path van een directory

Deze namen verschillen nog wel eens, maar **binnen** een project moet je consistent zijn. Als je bijvoorbeeld relatieve en absolute namen hebt **moet** het duidelijk zijn. Ik heb in de uploadserver daar bugs mee gezien.
Als het onderscheid niet optreedt: dan hoeft het ook niet in de variabelenaam :-)